packages/autocomplete-js/src/render.tsx

```diff
@@ -109,7 +109,9 @@ export function renderPanel<TItem extends BaseItem>(
         </div>
       )}
 
-      {items.length === 0 && source.templates.noResults && state.query ? (
+      {!items.length &&
+      source.templates.noResults &&
+      (state.query || state.isOpen) ? (
```

Comment:
We shouldn't need to check `state.query` anymore (removing the check doesn't break tests).

This check seems to be enough:

```ts
source.templates.noResults && items.length === 0 && state.isOpen
```

Reply:
I just realized that we don't even need to check if `isOpen` since we return early https://github.com/algolia/autocomplete/blob/5921f9b2d7b6a946f3fd7f8558ae1147fe70340f/packages/autocomplete-js/src/render.tsx#L77-L83

Fixed in https://github.com/algolia/autocomplete/pull/573/commits/bfb80ebd4d9bf719923e5f21b1aa1e3264c412ea